libsyclinterface/helper/source/dpctl_utils_helper.cpp

```diff
@@ -242,9 +251,13 @@ std::string DPCTL_AspectToStr(aspect aspectTy)
 aspect DPCTL_StrToAspectType(const std::string &aspectTyStr)
 {
     aspect aspectTy;
-    if (aspectTyStr == "host") {
+    if (false) {
```

Comment:
Why not just move the "cpu" check on top?

Reply:
Good point